server/service/GithubService.js

```diff
@@ -130,27 +131,28 @@ export default class GithubService {
     this.fetchPath('POST', path, payload, accessToken)
   }
 
+
   async readZapprFile(user, repo, accessToken) {
```

Comment:
While we're at it we could rename the `user` parameter to `owner` for reduced mental overhead.